src/tinycl.h

```diff
@@ -67,6 +67,10 @@ int clReleaseKernel(cl_kernel);
 cl_mem clCreateBuffer(cl_context, cl_mem_flags, size_t, void *, int *);
 int clReleaseMemObject(cl_mem);
 cl_command_queue clCreateCommandQueueWithProperties(cl_context, cl_device_id, const cl_queue_properties *, int *);
+#ifdef __APPLE__
+cl_command_queue clCreateCommandQueueWithPropertiesAPPLE(cl_context, cl_device_id, const cl_queue_properties *, int *);
```

Comment:
`cl_queue_properties` has extra flags, but it shouldn't matter if we are not using them.